modules/apps/commerce/commerce-discount-service/src/main/java/com/liferay/commerce/discount/service/impl/CommerceDiscountLocalServiceImpl.java

```diff
@@ -1906,6 +1870,62 @@ private Predicate _toTargetPredicate(
 		return predicate.withParentheses();
 	}
 
+	private void _validate(
+			long commerceDiscountId, long companyId, String couponCode,
+			BigDecimal level1, BigDecimal level2, BigDecimal level3,
+			BigDecimal level4, String limitationType,
+			BigDecimal maxDiscountAmount, String target, String title,
+			boolean useCouponCode)
```

Comment:
Try to sort the variable order like CommerceDiscount service.xml.

```suggestion
	private void _validate(
			long commerceDiscountId, long companyId, String title,
			String target, boolean useCouponCode, String couponCode,
			BigDecimal maxDiscountAmount, BigDecimal level1, BigDecimal level2,
			BigDecimal level3, BigDecimal level4, String limitationType)
```